agent-install/agent-install.sh

```diff
@@ -706,7 +706,7 @@ function process_node(){
 			# with a pattern currently
 			log_notify "The node currently has workload(s) (check them with hzn agreement list)"
 			if [[ -z "$HZN_EXCHANGE_PATTERN" ]] && [[ -z "$HZN_NODE_POLICY" ]]; then
-				log_info "Neither a pattern nor node policy has not been specified"
+				log_info "Neither a pattern nor node policy has been specified"
```

Comment:
Actually, you can register without pattern and without policy. The simple 'hzn register' command will cause your node to register with just the default builtin policy values.  Both -p and --policy are optional, but mutually exclusive.